Velaptor/AppStats.cs

```diff
@@ -156,7 +156,7 @@ internal static void RemoveLoadedTexture(uint textureId)
     /// <param name="fontInfo">The loaded font info.</param>
     internal static void RecordLoadedFont(string fontInfo)
     {
-        if (string.IsNullOrEmpty(fontInfo) is not false || !fontInfo.Contains("|size:"))
```

Comment:
@ljdarj Good catch on this one!!

Funny how things like this make it into code.  😉